.clasp.json

```diff
@@ -1,4 +1,4 @@
 {
-  "scriptId": "1N22pZ29WY4DF-KV-JZ7jLh92f44pWrgL0rUqLviBbQ8ECZCEMbpvxi9V",
+  "scriptId": "1wjCZWISevP9KgxxiNM9OPP9Aq3VnGIxVfS6BTj586WhfrKWTg8rRmvgq",
```

Comment:
you probably accidentally committed this, can you remove this from the pr?

Reply:
Should we just ignore this file?